guillotina/schema/tests/test_schema.py

```diff
@@ -13,43 +13,47 @@
 ##############################################################################
 # flake8: noqa
 import unittest
+from zope.interface.interface import InterfaceClass
 
 
 def _makeSchema():
     from zope.interface import Interface
     from guillotina.schema import Bytes
 
-    class ISchemaTest(Interface):
-        title = Bytes(
-            title="Title",
-            description="Title",
-            default=b"",
-            required=True)
-        description = Bytes(
-            title="Description",
-            description="Description",
-            default=b"",
-            required=True)
-        spam = Bytes(
-            title="Spam",
-            description="Spam",
-            default=b"",
-            required=True)
-    return ISchemaTest
+    return InterfaceClass(
```

Comment:
Do we need to change in all the interfaces defined inside code ???

Reply:
Just nested class defined interfaces might cause this it seems right now. mypy's integration throws an error on this. The reason is that it can't resolve the reference. Nested class defined interfaces don't have a proper `__module__`.  The `InterfaceClass` variant is identical and does proper `__module__`.

In my opinion, the `InterfaceClass` approach also looks nicer. I'm not a fan of using classes defined inside functions that use the function's closure.

Reply:
👍